src/choose-location/choose-location.component.tsx

```diff
@@ -41,42 +55,41 @@ export default function ChooseLocation(props: ChooseLocationProps) {
   const RadioInput = (option: RadioInputOption) => (
     <React.Fragment key={option.uuid}>
       <div className="omrs-radio-button">
-        <label>
-          <input
-            type="radio"
-            name="location"
-            value={option.uuid}
-            onChange={evt => setLocation(evt.target.value)}
-            ref={locationInputRef}
-            className={`omrs-margin-8`}
-          />
-          <span>{option.display}</span>
-        </label>
+        <input
+          id={option.uuid}
+          type="radio"
+          name="location"
+          value={option.uuid}
+          onChange={evt => setLocation(evt.target.value)}
+          ref={locationInputRef}
+          className={`omrs-margin-8`}
+        />
+        <label htmlFor={option.uuid}>{option.display}</label>
       </div>
     </React.Fragment>
   );
 
   return (
     <div className={`canvas ${styles["container"]}`}>
       <h1>
-        <Trans i18nKey="welcome">Welcome!</Trans>
+        <Trans i18nKey="welcome">Welcome {currentUser}</Trans>
       </h1>
       <form onSubmit={handleSubmit} ref={formRef}>
-        <div className={`omrs-card ${styles.card}`}>
+        <div className={`${styles["location-card"]} omrs-card`}>
```

Comment:
nit: it's convention to name classes using camelCase and access them with the dot notation instead e.g `styles.locationCard` 🙂

Reply:
Thanks @fatmali .  I was wondering about this.  I have never seen any styles named with camel case, they all seem to use hyphens to separate words.  Are these hyphens automatically converted to camel case for usage?  What I did here was just following existing patterns that were on the page.  Can you point me to any documentation that describes how developers should name and access custom styles?  That would really help.

Reply:
@mseaton it's more of a preference rather than a convention. The additional `[]` and `-` is considered verbose in the react community. I can't find any articles to support this, so I guess we can let it stay for now as there are existing patterns already.